3.6.1/entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+log(){
+	while read line ; do
+		echo "`date '+%D %T'` $line"
+	done
+}
+
+set -e
+logfile=/home/LogFiles/entrypoint.log
+test ! -f $logfile && mkdir -p /home/LogFiles && touch $logfile
+exec > >(log | tee -ai $logfile)
+exec 2>&1
+
+set_var_if_null(){
+	local varname="$1"
+	if [ ! "${!varname:-}" ]; then
+		export "$varname"="$2"
+	fi
+}
+
+python --version
+pip --version
+
+echo "INFO: starting SSH ..."
+service ssh start
+
+# setup nginx log dir
+test ! -d "$NGINX_LOG_DIR" && echo "INFO: $NGINX_LOG_DIR not found, creating ..." && mkdir -p "$NGINX_LOG_DIR"
```

Comment:
We should look at not having a hard dependency on /home for logging. Instead if we just make sure everything we need to get logged goes to stdout and stderr that will be sufficient as we will acquire this as part of docker logs